pennylane/_qubit_device.py

```diff
@@ -311,6 +316,17 @@ def execute(self, circuit, **kwargs):
         Returns:
             array[float]: measured value(s)
         """
+
+        if logger.isEnabledFor(logging.DEBUG):
+            logger.debug(
+                "Entry with args=(circuit=%s, kwargs=%s) called by=%s",
+                circuit,
+                kwargs,
+                "::L".join(
+                    str(i) for i in inspect.getouterframes(inspect.currentframe(), 2)[1][1:3]
+                ),
+            )
```

Comment:
IIRC there was something about f-strings not being recommended?

Reply:
The issue is the docs for logging were written prior to f-string support in Python. With this, no docs use f-strings, and to be consistent with the docs I adopted their style through. We can modify this, but I think it is better for consistency with the docs for now, and if required we can upgrade later.